tests/ecdhp256-test.c

```diff
@@ -89,8 +101,75 @@ int main()
 	ok = ok && compare_and_print(64, point_compressed, compressed7);
 
 
+// The start of Wycherproof tests
+
+	printf("%s\n", "Wycheproof tests ECDH: ");
+		
+	for (int i = 0 ; i< sizeof(w_vectors)/sizeof(ecdhp256_w_i); i++)
+	{
+		printf("ECDH Wycheproof Test %d \n", i );
+
+		bool flagDecompressSuccessful = Hacl_P256_decompression_not_compressed_form(w_vectors[i].publicKey, decompressedPoint);
+		uint64_t success = Hacl_P256_ecp256dh_r(result, decompressedPoint, w_vectors[i].privateKey);
+		
+		if (w_vectors[i].flag != 1) 
```

Comment:
what does  		`if (w_vectors[i].flag != 1)` indicate? I'm not familiar with the wycheproof test data

Reply:
sometimes you do != 1 and in some other places you compare the value of the flag with UINT64_MAX or 0 so I'm not sure what this is supposed to stand for